pkg/api/types.go

```diff
@@ -1911,6 +1909,8 @@ type PodSpec struct {
 	// If not specified, the pod will be dispatched by default scheduler.
 	// +optional
 	SchedulerName string
+	// List of initialization containers belonging to the pod.
+	InitContainers []Container
```

Comment:
@smarterclayton new ID of json field has to be assigned. I found this approach least impacting, since ID's have to be sorted - I didn't intend to influence other fields.